### PR TITLE
{tools}[foss/2018b] TensorFlow v1.10.1 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

edit: requires https://github.com/easybuilders/easybuild-easyblocks/pull/1486

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3c52ff81ceb839b3d5e5675c80a6220d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3126.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/0129317f599e05ea18110deef50d532f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node2097.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/df5a874a87a9d997c08e620ee2d16aa7 for a full test report.

File: easybuild/easyconfigs/w/wheel/wheel-0.31.1-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description = """A built-package format for Python."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['0a2e54558a0628f2145d2fc822137e322412115173e8a2ddbe1c9024338ae83c']
+
+dependencies = [('Python', '3.6.6')]
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/wheel/wheel-0.31.1-foss-2018b-Python-3.6.6.eb]: Shouldn't this one also use
download_dep_fail = True
use_pip = True

like you told me to use for Inelastica?

Comment (boegel) [reply easybuild/easyconfigs/w/wheel/wheel-0.31.1-foss-2018b-Python-3.6.6.eb]: Yes, it should, added that (consistently) in ff7dbe402

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e38eb5f5454ff26435dee91b7db6b7e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (3 easyconfigs in this PR)
node2610.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0f5a92e509aa1fce34208ee3f7a0c434 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2043.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d4102f5265745d6c97c36a4421058a79 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (3 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3709b9e6b0faf0a9be2e3e7a9cb29dce for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2106.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/627c2181fdcf03685b2f2b30b03a393d for a full test report.

**Pull request merged**